rclcpp/test/rclcpp/node_interfaces/test_node_graph.cpp

```diff
@@ -64,6 +64,11 @@ class TestNodeGraph : public ::testing::Test
     rclcpp::shutdown();
   }
 
+  static void TearDownTestCase()
+  {
+    rclcpp::shutdown();
+  }
```

Comment:
This one doesn't seem right.  Here we are calling `rclcpp::init` in `SetUp`, so either `rclcpp::shutdown` in `TearDown` should be enough, or we should split `rclcpp::init` into `SetUpTestCase`.

Reply:
https://github.com/ros2/rclcpp/pull/2633/commits/9af7a4d149f7b7842a47242097222560bd4ec141